src/main/java/org/alliancegenome/curation_api/jobs/executors/ParalogyExecutor.java

```diff
@@ -48,7 +48,7 @@ public void execLoad(BulkLoadFileHistory bulkLoadFileHistory) {
 			bulkLoadFileHistory.setCount(paralogyData.getData().size());
 			updateHistory(bulkLoadFileHistory);
 
-			boolean success = runLoad(geneToGeneParalogyService, bulkLoadFileHistory, dataProvider, paralogyData.getData(), paralogyIdsLoaded, false);
+			boolean success = runLoad(geneToGeneParalogyService, bulkLoadFileHistory, dataProvider, paralogyData.getData(), paralogyIdsLoaded, false, "Records");
```

Comment:
Shouldn't need to pass in "Records" here

Reply:
Done